MilitaryManager.Units.API/Controllers/AuditValueDetalesController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using MilitaryManager.Core.Interfaces.Services;
+using System.Threading.Tasks;
+
+namespace MilitaryManager.Units.API.Controllers
+{
+
+    [ApiController]
+    [Route("api/[controller]")]
+    public class AuditValueDetalesController : Controller
```

Comment:
I suppose, here is typo, and you meant 'AuditValue**Details**Controller ' instead of 'AuditValue**Detales**Controller '